lib/babel-plugin-dynamic-import-node.d.ts

```diff
@@ -0,0 +1,16 @@
+import type { PluginAPI } from "@babel/core";
+import type * as t from "@babel/types";
+// https://github.com/airbnb/babel-plugin-dynamic-import-node/blob/master/src/utils.js
```

Comment:
When I combine the declaration here with `third-party-libs.d.ts`. TS seems to be confused and resolve `"babel-plugin-dynamic-import-node/utils"` as `{ default: plugin }` like the other Babel plugins in `third-party-libs.d.ts`. I am not sure if the behaviour is intended or a TS bug.